pkg/investigations/registry.go

```diff
@@ -30,9 +30,19 @@ var availableInvestigations = []investigation.Investigation{
 // linked to single alert type.
 func GetInvestigation(title string, experimental bool) investigation.Investigation {
 	for _, inv := range availableInvestigations {
-		if inv.ShouldInvestigateAlert(title) && (experimental || !inv.IsExperimental()) {
+		if strings.Contains(title, inv.AlertTitle()) && (experimental || !inv.IsExperimental()) {
 			return inv
 		}
 	}
 	return nil
 }
+
+// GetAvailableInvestigationsTitles returns a string array with the alert titles of all available investigations.
+func GetAvailableInvestigationsTitles() []string {
```

Comment:
This seems to be dead code?
I don't see this being used anywhere.

Reply:
Good catch, I added that to facilitate the new e2e test I want to add in https://github.com/openshift/configuration-anomaly-detection/pull/542
Arguably it should be moved to the other PR, but if okay for you, I will keep it in this one 

 https://github.com/openshift/configuration-anomaly-detection/pull/542/files#diff-b41f8eaba3f873a581795129bfc035cd80d855da0564a2016c07aa0e3a16c439R88